### ISSUE TITLE
Cloning a pilot causes all key bindings to be reset

### ISSUE THREAD

**Issue by Goober5000** [scp-fs2open/fs2open.github.com#1974]
In retail, cloning a new pilot from an existing pilot copies all of the key bindings.  In the latest FSO, all the key bindings are cleared.

**Comment by FSCyborg** [comment]
I just attempted this with the GOG exe and it actually doesn't copy the key bindings.  Also, Git blame seems to show that the code for cloning has been the same since retail.

**Comment by Goober5000** [comment]
I tested this with the original retail EXE when I opened this ticket.  For sanity's sake I just went back and tested it again -- all key bindings are preserved.  I also tested it with a FSO build based on the original public source code release, and it preserved all the key bindings too.

**Comment by FSCyborg** [comment]
This problem may be way worse than we thought.

After looking through the code, I could find no difference in functionality for the intro screen cloning.  In fact, I tested it and it worked perfectly.

To be complete, I went over to the barracks, and ... well.... it is borked.  Any pilot created through the barracks has no key bindings, whether original or cloned.  I am going to look through the barracks code now to see if I can find the cause.

**Comment by FSCyborg** [comment]
So, I haven't found the code that is causing this, however, I have some interesting insights and will be gone for the next few days.

I found out exactly when it happens in game, though.  It is not the Clone button or the pilot create button, and it is not the Select Pilot button towards the top of the screen.  It is once you go from the barracks to the mainhall.  I confirmed this by using Alt F4 at various times.
